src/App.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import axios from "axios";
+
+//image
+import shoesCOMME from "./img/shoesCOMME.png";
+import shoesFire from "./img/shoesFIre.png";
+import shoesBlack from "./img/shoesBlack.png";
+import shoesFireBlack from "./img/shoesFireBlack.png";
+import shoesCamo from "./img/shoesCamo.png";
+import shoesGreen from "./img/shoesGreen.png";
+import shoesBlue from "./img/shoesBlues.png";
+import shoesCons from "./img/shoesCons.png";
+import shoesRed from "./img/shoesRed.png";
+import shoesRed01 from "./img/shoesRed01.png";
+import shoesPink01 from "./img/shoesPink01.png";
+import shoes01 from "./img/shoes01.png";
+import shoesBas from "./img/shoesbas.png";
+import shoesGaru from "./img/shoesGaru.png";
+import shoes03 from "./img/shoes03.png";
+
+//Components
+import Allshoes from "./allshoes";
+import White from "./White";
+
+export default function App() {
+  const [allshoesName, setallShoesName] = useState([]);
```

Comment:
not sure what `Name` means here, also `allShoes` capitalization can be changed

```suggestion
  const [allShoes, setAllShoes] = useState([]);
```